ios/brave-ios/App/Client.xcodeproj/xcshareddata/xcschemes/Nightly.xcscheme

```diff
@@ -1,10 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
    LastUpgradeVersion = "1250"
-   version = "1.3">
+   version = "1.7">
    <BuildAction
       parallelizeBuildables = "YES"
       buildImplicitDependencies = "YES">
+      <PreActions>
+         <ExecutionAction
+            ActionType = "Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction">
+            <ActionContent
+               title = "BraveCore Pre-actions"
+               scriptText = "sh &quot;${PROJECT_DIR}/../scripts/xcode_scheme_preaction.sh&quot;&#10;">
```

Comment:
I assume `&quot` characters are expected to be here

Reply:
Yes, Xcode scheme editor is writing this file